### ISSUE TITLE
[Feature Request] - Tree structure

### ISSUE THREAD

**Issue by alx-xlx** [KoalaBear84/OpenDirectoryDownloader#22]
**Is your feature request related to a problem? Please describe.**
Surfing through large amounts of data is painful as the page loads each time we navigate through directories
**Describe the solution you'd like**
[snap2html - Example](https://www.rlvision.com/snap2html/example.html) this is what I want to achieve, being able to generate an html file with Tree Structure of all the directories and files.
Thus making it easier to search.
[Source Code](https://github.com/rlv-dan/Snap2HTML)

**Comment by KoalaBear84** [comment]
Yes, that is a nice feature. I tested Snap2HTML, it also is struggling with more than 100.000 files or so. I fails without warning.

I'm not sure if I have the time to create anything like this anytime soon.

**Comment by alx-xlx** [comment]
I found this script and made a simple Google Colab script to quickly build the tree structure of Google Drive
https://github.com/alx-xlx/gdrive-index

**Comment by alx-xlx** [comment]
I just wanted to know if the reason behind failing to index 1,00,000+ files could be the bulky html file

**Comment by KoalaBear84** [comment]
I guess it's too big for Snap2HTML.

**Comment by KoalaBear84** [comment]
There was an improvement 14 hours ago on memory usage 😂

https://github.com/rlv-dan/Snap2HTML/commit/aeb8ec32a17d955ede26484aa5002022a4f0a3e0
